apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,126 @@
+import { Client } from "auth0";
+
+const USER_ID = "google-oauth2|115545703201865461059";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let newClientId: string;
+
+  const getToken = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt")
+      .then((token: string) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getToken();
+    if (!accessToken) {
+      cy.loginByAuth0Api();
+      cy.wait(1500);
+      getToken();
+    }
+  });
+
+  it("deletes the client_id from user metadata", () => {
```

Comment:
These api signatures don't look like createMySecret/Client, getMySecret/Client, and deleteMySecret/Client.

If the api accepts any way through path paramters, query parameters, or the body to specify which client or user we want the client for, then we have security vulnerabilities. The api needs to grab whatever identifying information it needs from the access token, and only use that to determine which client to access.